NewsApp/NewsApp/Controller/ThematiqueController.swift

```diff
@@ -9,21 +9,33 @@
 import UIKit
 
 class ThematiqueController: UITableViewController {
-
-    private var thematique: [String] = ["Économie", "Divertissement", "Santé", "Sports", "Sciences", "Technologie"]
-
+    
+    private let category = ["Économie", "Sports", "Santé", "Science", "Divertissement", "Technologie"]
     override func viewDidLoad() {
         super.viewDidLoad()
     }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "segueToNews" {
+            guard let newsController = segue.destination as? NewsController else {
+                return
+            }
+            guard let indexPath = sender as? IndexPath else {
+                return
+            }
+            newsController.title = category[indexPath.row]
+            newsController.category = Category.allCases[indexPath.row]
+        }
+    }
 }
 
 extension ThematiqueController {
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return thematique.count
+        return category.count
     }
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if let cell = tableView.dequeueReusableCell(withIdentifier: "ThematiqueCellIdentifier", for: indexPath) as? ThematiqueCell {
```

Comment:
Dans l'idéal tu peux également ajouter les identifiers des cells dans le fichier de constantes également

Reply:
fix in "fix PR"